pkg/pod/pod.go

```diff
@@ -262,3 +257,33 @@ func MakeLabels(s *v1alpha1.TaskRun) map[string]string {
 	labels[taskRunLabelKey] = s.Name
 	return labels
 }
+
+// getLimitRangeMinimum gets all LimitRanges in a namespace and
+// searches for if a container minimum is specified. Due to
+// https://github.com/kubernetes/kubernetes/issues/79496, the
+// max LimitRange minimum must be found in the event of conflicting
+// container minimums specified.
+func getLimitRangeMinimum(namespace string, kubeclient kubernetes.Interface) (corev1.ResourceList, error) {
+	limitRanges, err := kubeclient.CoreV1().LimitRanges(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	min := allZeroQty()
+	for _, lr := range limitRanges.Items {
+		lrItems := lr.Spec.Limits
+		for _, lrItem := range lrItems {
+			if lrItem.Type == corev1.LimitTypeContainer {
+				if lrItem.Min != nil {
+					for k, v := range lrItem.Min {
```

Comment:
Open to not having this approach, which is to search for the max LimitRange container min in the event there are conflicting container mins. 

An alternative approach would just be to break the loop after the first container min is found and let it potentially fail.